packages/app/src/pages/endpoints/[endpointId]/_/content/parts/action/index.tsx

```diff
@@ -42,10 +43,22 @@ const Action: React.FC<Props> = ({ request, onClick }) => {
     onClick();
   }, [onClick]);
 
+  const { language } = useI18n();
+
   const label = useMemo<string>(() => {
     const { operation } = request;
-    if (operation.summary) {
-      return operation.summary;
+    switch (language) {
+      case 'en':
+        if (operation.summary) {
+          return operation.summary;
+        }
+        break;
+      case 'ja':
+        if (operation.description) {
+          return operation.description;
+        } else if (operation.summary) {
+          return operation.summary;
+        }
```

Comment:
Why do you need separate implementations for each language?

Reply:
@nonoakij 
In this part, it seems that the **"summary"** is obtained by referring to the data in `/example/golang/openapi/yaml/`

```yaml
  /users/{userId}:
    put:
      operationId: updateUser
      tags:
        - user
      summary: update user
      description: ユーザー更新
      parameters:
        - $ref: '#/components/parameters/UserIdPathParam'
      requestBody:
        required: true
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/UserUpdatePayload'
      responses:
        204:
          description: OK
      security:
        - jwt: []
```

The Japanese item looked like **"description"**, so I branched out to get that data, and if **"description"** didn't exist, I implemented it to get **"summary"** as before.

Reply:
> The Japanese item looked like "description"

`description` in schema provided by you is in Japanese, but it does not need to be in Japanese.It is used to explain in more detail what the operation does.

Is it possible to remove splitting by language?